tests/bats/settings/security.bats

```diff
@@ -7,12 +7,12 @@ load ../_helpers_govcms
 setup() {
   if [ ! -f "/tmp/bats/all.settings.php" ]; then
     mkdir -p /tmp/bats
-    (cd /tmp/bats && curl -O https://raw.githubusercontent.com/govcms/scaffold-tooling/develop/drupal/settings/all.settings.php)
+    cp ./drupal/settings/all.settings.php /tmp/bats/
```

Comment:
Not sure why these were downloading from github - seems like a weird decision testing features against settings from develop which might have changed on the feature?...

Oh well changed now :D